### ISSUE TITLE
Support tags in extension

### ISSUE THREAD

**Issue by FWDavide** [xmollv/Llitgi#58]
When using the extension for submitting a new link allow the user to select/create one or multiple tags.

**Comment by xmollv** [comment]
Yep, I agree that we should have that. IDK how tricky it will be due to having to interact with Core Data from an extension. The first step will be to enable adding new tags (and removing them) from the app itself.
